collect_app/src/main/java/org/odk/collect/android/fragments/dialogs/FirstLaunchDialog.kt

```diff
@@ -0,0 +1,43 @@
+package org.odk.collect.android.fragments.dialogs
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.appcompat.widget.Toolbar
+import androidx.fragment.app.DialogFragment
+import kotlinx.android.synthetic.main.first_launch_dialog_layout.*
+import org.odk.collect.android.R
+import org.odk.collect.android.activities.ActivityUtils
+import org.odk.collect.android.activities.MainMenuActivity
+import org.odk.collect.material.MaterialFullScreenDialogFragment
+
+class FirstLaunchDialog : MaterialFullScreenDialogFragment() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setStyle(DialogFragment.STYLE_NO_TITLE, R.style.Theme_Collect_Light)
```

Comment:
This particular line is removed but generally the activity uses a theme that extends `Theme.Collect.BaseLight`. I think that's ok because it's our default them and if you install the app for the first time to display that dialog that light them should be used.